packages/react/src/components/Storage/File-Uploader/UploaderButton/index.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React from 'react';
+import { Button, VisuallyHidden } from '../../../../primitives';
+import { UploaderButtonProps } from '../types';
+import { translate } from '@aws-amplify/ui';
+
+export function UploaderButton({
+  multiple,
+  acceptedFileTypes,
+  onUpload,
+  className,
+}: UploaderButtonProps): JSX.Element {
```

Comment:
```suggestion
}: UploaderButtonProps) {
```
as it can be auto-inferred.

Reply:
OK

Reply:
Actually, my editor is saying it's missing a return type, so I"ll add it back in

Reply:
sgtm